pkg/webhook/cluster/nutanix_validator.go

```diff
@@ -125,3 +134,33 @@ func checkIfPrismCentralIPInLoadBalancerIPRange(
 
 	return nil
 }
+
+// checkIfPrismCentralAndControlPlaneIPSame checks if Prism Central and Control Plane IP are same.
+// It compares strictly IP addresses(no FQDN) and doesn't involve any network calls.
+// This is a temporary check until we have a better way to handle this by reserving IPs
+// using IPAM provider.
```

Comment:
I would suggest we remove this and not have include predictions.
```suggestion
```

Reply:
21293e9ffa7fd596f3ba894845b1c7f3f74f177c